### ISSUE TITLE
Map not displaying on Android

### ISSUE THREAD

**Issue by shivapersad** [Yermo/nativescript-mapbox#382]
TNS Version: 6.8.0
nativescript-mapbox version: 4.4.1
Platform: Android

package.json contents:

"nativescript": {
    "id": "org.ngccng.cngApp",
    "tns-ios": {
      "version": "6.5.2"
    },
    "tns-android": {
      "version": "6.5.3"
    }
  },
  "dependencies": {
    "@angular/animations": "~8.2.0",
    "@angular/common": "~8.2.0",
    "@angular/compiler": "~8.2.0",
    "@angular/core": "~8.2.0",
    "@angular/forms": "~8.2.0",
    "@angular/platform-browser": "~8.2.0",
    "@angular/platform-browser-dynamic": "~8.2.0",
    "@angular/router": "~8.2.0",
    "nativescript-angular": "~8.20.0",
    "nativescript-drop-down": "^5.0.2",
    "nativescript-fancyalert": "^3.0.8",
    "nativescript-iqkeyboardmanager": "^1.5.1",
    "nativescript-mapbox": "^4.4.1",
    "nativescript-theme-core": "~1.0.2",
    "reflect-metadata": "~0.1.12",
    "rxjs": "^6.4.0",
    "rxjs-compat": "^6.5.5",
    "tns-core-modules": "~6.5.0",
    "validator": "^13.1.1",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~8.2.0",
    "@ngtools/webpack": "~8.2.0",
    "codelyzer": "~4.5.0",
    "nativescript-dev-webpack": "~1.5.1",
    "node-sass": "^4.7.1",
    "tslint": "~5.19.0",
    "typescript": "~3.5.3",
    "babel-traverse": "6.25.0",
    "babel-types": "6.25.0",
    "babylon": "6.17.4",
    "lazy": "1.0.11"
  }
}

When I attempt to display a map using the plugin on Android I get the following error:

**JS: ERROR Error: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference**

Any assistance is appreciated. Thanks.

**Comment by Yermo** [comment]
Does the demo work?

**Comment by shivapersad** [comment]
Hi Yermo,

Apologies for the delayed response. I reviewed my code against your documentation and saw that I needed to update my structure as I was utilizing legacy functions from a previous version of the plugin. Once I updated the structure everything worked perfectly. I have both iOS and Android versions working properly. As such I am closing this issue. Thanks again for your work on this plugin.
